sql/core/src/main/scala/org/apache/spark/sql/execution/command/commands.scala

```diff
@@ -237,15 +237,18 @@ case class ExplainCommand(
     logicalPlan: LogicalPlan,
     override val output: Seq[Attribute] =
       Seq(AttributeReference("plan", StringType, nullable = true)()),
-    extended: Boolean = false)
+    extended: Boolean = false,
+    codegen: Boolean = false)
```

Comment:
Sure. I'll change that. By the way, I think you mean (1). Am I right?
(1) `EXPLAIN CODEGEN SELECT ~`
(2) `CODEGEN SELECT ~`